classes/fullstack/project1/server/src/providers/mongodb.ts

```diff
@@ -0,0 +1,18 @@
+import { MONGODB_DATABASE_NAME, MONGODB_URI } from '../config'
```

Comment:
Added logging as suggested.
Moved logging before "await connect" (otherwise first connection will not be logged).

If there are any issues with ts support, I need more details. For now I see no typescript errors or any types.